webview-ui/src/App.tsx

```diff
@@ -60,6 +61,11 @@ function App() {
   window.addEventListener("message", (event) => {
     const message = event.data as ExtensionToWebviewMessage;
     switch (message.messageType) {
+      case "setLoading": {
+        const params = message.params as SetLoadingParams;
+        setLoadingMessage(params.loading);
```

Comment:
Again, this one is on me. When I changed it to use the loading spinner instead of loading message text, the `setLoadingMessage` and associated `loadingMessage` variable would probably make more sense as `isLoading` or `showLoadingIndicator` or something. It could make sense as "the message is loading" -> `loadingMessage` I guess.

Reply:
Not sure what you want me to do here?

Reply:
Yeah I don't think what I said made that much sense lol don't worry about it